src/inverseRobot.js

```diff
@@ -4,10 +4,20 @@
  * @param {object} robot
  *
  * @return {object}
-*/
+ */
 
 function inverseRobot(robot) {
-  // write code here
+  let newRobot = {};
+
+  for (const [key, value] of Object.entries(robot)) {
```

Comment:
You can create a variable with entries separately and use it in your loop;

Reply:
let's leave it up to the mentors